config/uitests-amo.js

```diff
@@ -0,0 +1,17 @@
+module.exports = {
+  // For use on addons-server, or with the command `amo:ui-tests` and the
+  // HOSTNAME  variable as `ui-tests`.
+  apiHost: 'http://olympia.test',
+  proxyApiHost: 'http://olympia.test',
+  proxyPort: 3000,
+  proxyEnabled: true,
+  // Setting this to false returns add-ons that are not compatible but means
+  // developers can pull from a much larger dataset on the local/-dev/-stage
+  // servers. Set this to true to only get compatible add-ons (this is what
+  // happens in production) but get a lot fewer add-ons in search results.
+  restrictSearchResultsToAppVersion: false,
+  fxaConfig: 'default',
+  trackingEnabled: false,
+  loggingLevel: 'debug',
+  enableAMInstallButton: true,
+};
```

Comment:
Don't you need `enableNodeStatics: true,` here too? I see it used in different "test-like" envs.

Reply:
You can specify `CSP: false` here too.